src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+    Hello, world!
+
+  </body>
+</html>
```

Comment:
Add an empty line at the end
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file
![image](https://user-images.githubusercontent.com/96409563/220557362-3c2c34a8-8b49-4be4-9fb7-e1ceb1cd3c50.png)

Reply:
> Add an empty line at the end https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file ![image](https://user-images.githubusercontent.com/96409563/220557362-3c2c34a8-8b49-4be4-9fb7-e1ceb1cd3c50.png)

I'm corrected my files css and html. Check my work please))